templates/macros/translator.html

```diff
@@ -0,0 +1,9 @@
+{%- macro translator(text, bag_name, language, fallback='de') %}
+  {%- if bag(bag_name, language, text) -%}
+    {{ bag(bag_name, language, text) }}
+  {%- elif bag(bag_name, fallback, text) -%}
+    {{ bag(bag_name, fallback, text) }}
+  {%- else -%}
+    {{ text }}
+  {%- endif %}
+{%- endmacro %}
```

Comment:
den tranlator find ich cool. nur man könnte ihn in .j2 umbennen. egal. mach halt nicht.

Reply:
wie gesagt eine Umbenennung in j2 kann man da durchaus machen. Einige der anderen Macros die von dir stammen enden auch mit .html und ich habe mich daher einfach daran gehalten wie das bisher war.

Reply:
umbenannt